config/software/nodejs.rb

```diff
@@ -19,7 +19,7 @@
 license "MIT"
 license_file "LICENSE"
 
-if ppc64? || ppc64le?
+if ppc64? || ppc64le? || ohai["kernel"]["machine"] == "s390x"
```

Comment:
This shouldn't block merge, but I would consider making this conditional into its own method in omnibus or some such.